src/main/java/uk/gov/hmcts/reform/professionalapi/controller/SuperController.java

```diff
@@ -250,7 +250,7 @@
             jurisdictionService.propagateJurisdictionIdsForSuperUserToCcd(professionalUser, userId);
             ResponseEntity<Object> responseEntity = createUserProfileFor(professionalUser, null, true,
                     false);
-            if (responseEntity.getStatusCode().is2xxSuccessful()) {
+            if (null != responseEntity.getBody() && responseEntity.getStatusCode().is2xxSuccessful()) {
```

Comment:
better to check responseEntity.getStatusCode().is2xxSuccessful() first and then body

Reply:
Addressed here: https://github.com/hmcts/rd-professional-api/pull/756/commits/269e0cdf95e711f02eabdfc64518cd59007083df